TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/RoutesToProfessionalStatus/AddRoute/TrainingProvider.cshtml

```diff
@@ -0,0 +1,20 @@
+﻿@page "/route/add/training-provider/{handler?}"
+@model TeachingRecordSystem.SupportUi.Pages.RoutesToProfessionalStatus.AddRoute.TrainingProviderModel
+
+@section BeforeContent {
+    <govuk-back-link href=@(Model.BackLink)>Back</govuk-back-link>
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+        <form action="@LinkGenerator.RouteAddTrainingProvider(Model.PersonId, Model.JourneyInstance!.InstanceId, Model.FromCheckAnswers)" method="post">
+            <span class="govuk-caption-l">Routes and professional status - @Model.PersonName</span>
+
```

Comment:
Just a stub page at the moment, for the benefit of testing the journey after end-date page